include/datadog/opentracing.h

```diff
@@ -63,26 +63,26 @@ class TraceEncoder {
   virtual ~TraceEncoder() {}
 
   // Returns the Datadog Agent endpoint that traces should be sent to.
-  virtual const std::string &path() = 0;
+  virtual const std::string& path() = 0;
```

Comment:
Did clang-format change the rules again?

Reply:
I don't even know! I didn't do this on purpose though!